src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -611,9 +611,10 @@ private InternalBackupResult CreateLocalBackupOrSnapshot(PeriodicBackupStatus st
 
                         var totalSw = Stopwatch.StartNew();
                         var sw = Stopwatch.StartNew();
+                        var compressionAlgorithm = _configuration.SnapshotSettings?.CompressionAlgorithm ?? _database.Configuration.Backup.CompressionAlgorithm;
                         var compressionLevel = _configuration.SnapshotSettings?.CompressionLevel ?? CompressionLevel.Optimal;
```

Comment:
introduce here a config option for default compression level for snapshot backups, check if we need to introduce option for smuggler backups compression level as well

Reply:
done